aws-ec2-transitgatewayroutetable/src/main/java/software/amazon/ec2/transitgatewayroutetable/Translator.java

```diff
@@ -0,0 +1,128 @@
+package software.amazon.ec2.transitgatewayroutetable;
+
+
+
+import software.amazon.awssdk.services.ec2.model.*;
+
+import software.amazon.awssdk.utils.CollectionUtils;
+import java.util.Map;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Optional;
+
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import software.amazon.awssdk.services.ec2.model.TransitGatewayRouteTableState;
+
+public class Translator{
+
+    static DescribeTransitGatewayRouteTablesRequest translateToReadRequest(final String transitGatewayId){
```

Comment:
We should rename this argument to `transitGatewayRouteTableId`

Reply:
done thank you